docs/user/index.rst

```diff
@@ -0,0 +1,120 @@
+######################
+Using Aldryn Reversion
+######################
+
+After you set up and configured your models (they are registered for revisions
+support) you can start using ``aldryn-reversion`` end-user features.
+
+************************
+Restoring object version
+************************
+
+Most likely you already know that django keeps track of object changes also
+known as history, it is available on object admin edit form on the top of the
+page. With ``aldryn-reversion`` that history has more features.
+As soon as object is created - 'Initial version' is created and treated as
+restore point.
+
+Each record in history has a date which is now a link to restore revision
```

Comment:
How about:
"For each record in an object's history you'll now find a link to the 'restore revision' form."
?